language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -440,7 +455,7 @@ private static boolean withinTextRange(int position, NonTerminalNode node) {
             return Optional.empty();
         }
         String name = ((SimpleNameReferenceNode) referenceNode).name().text();
-        List<Symbol> visibleSymbols = context.visibleSymbols(context.getPosition());
+        List<Symbol> visibleSymbols = context.visibleSymbols(context.getCursorPosition());
```

Comment:
Same comment regarding the use of `visibleSymbols()` vs. `symbol()`

Reply:
Replied for the earlier comment